docs/docs/guides/local-development.md

```diff
@@ -94,6 +94,24 @@ pnpm run test:watch    # Run tests in watch mode
 pnpm run lint          # Lint code
 ```
 
+### Direct Client Chat UI
+
+```
+# Open a terminal and Start with specific character
+pnpm run dev --characters="characters/my-character.json"
+```
+```
+# Open a 2nd terminal and go to the client directory
+cd client
+pnpm install
+pnpm run dev
```

Comment:
Actually there was this `"start:client"` script https://github.com/ai16z/eliza/blob/main/package.json#L8 we can use to start the web UI for chat, but we need to update it to `pnpm --dir client dev` as there's no `start` script in `client/package.json`.

Reply:
I don't mind having something that works better - we just need to document it properly.  
Many users don't know how to start this easily and it gives them a great feel of accomplishment being able to launch the chat ui.